Rocket.Chat/Extensions/UIButton/UIButtonAccessibilityExtension.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIButtonAccessibilityExtension.swift
+//  Rocket.Chat
+//
+//  Created by Rudrank Riyam on 11/04/19.
+//  Copyright © 2019 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+extension UIButton {
+    func applyShowMoreButtonAccessibility() {
```

Comment:
I believe this method does not belong here. We have to be careful while extending a class (especially the ones provided by apple). `applyShowMoreButtonAccessibility` is not something that we can apply to any generic `UIButton`, so adding this method as an extension to `UIButton` is not the best idea. 

You should maybe consider moving this method to the controller.

Reply:
Thank you for your review. I understand your concern, and will work on it.
I have a solution in my mind right now, to replace the existing VOfunctions with a general method which takes the accessibilityLabel and returns the VOLocalizedString, and moving it to the controller